xmtp_mls/src/groups/validated_commit.rs

```diff
@@ -226,6 +228,11 @@ impl ValidatedCommit {
             &mutable_metadata,
         )?;
 
+        // Block any ReInit proposals
+        if staged_commit.psk_proposals().any(|_| true) {
```

Comment:
Should this be added to the comment on `ValidatedCommit` to list this check as well?